2025/1/29/how-i-ai.rst

```diff
@@ -0,0 +1,350 @@
+  ChatGPT, particularly on my phone.  I pay for the Plus subscription
+  because I use it enough to get a lot of value out of that.  A
+  significant use of this for me is in fact the voice mode, more on that
+  later.
+
+You can get most of the benefits of this from just paying for a single AI tool.
+I think as one expands the use, as one gets better at prompting it, the desire
+naturally grows to use more tooling.  As for which models to use day to
+day (and services) I don't have a particular strong strategy and
+preferences change quickly.  For instance since Deepseek's R1 came up, I'm
+exploring it quite a bit for programming because of how good a job it
+does and as of writing that's just a few days old.
+
+If you want to run models locally, Apple Silicon machines currently offer
+some of the best “bang for your buck” in terms of performance, power
+usage and money.  With `tailscale <https://tailscale.com/>`__, I can even
+access my MacBook's Open WebUI interface from my phone, as long as it is
+powered on.
+
+.. raw:: html
+
+    <details>
+      <summary>Guide for installing llm and Open WebUI</summary>
+
+This is a bit of a plug for `uv <https://docs.astral.sh/uv/>`__ quite
+honestly.  If you have it, it's crazy quick to get Open WebUI and ``llm``
+running locally::
+
+    uv tool install -p python3.11 open-webui
+    uv tool install --with llm-ollama --with llm-claude-3 --with llm-deepseek llm
+
+Afterwards you can launch `open-webui` and use the llm tool::
+
+    open-webui serve
+    llm keys set deepseek
+    llm -m deepseek-coder '# write timsort in python'
+
+* By default, Open WebUI only connects to OpenAI or Ollama. If you want to
+  add Anthropic or Deepseek, you need to create a custom function in the
+  “Functions” section of Open WebUI's admin panel.
+
+* Import the functions and configure your API keys, and you’re ready to go.
+
+Functions you can import to the WebUI:
+
+* `anthropic_manifold_pipeline.py <https://openwebui.com/f/justinrahb/anthropic>`__
+* `Deepseek <https://openwebui.com/f/xgawatt/DeepseekAPI>`__
+
+.. raw:: html
+
+    </details>
+
+AI Affordances
+--------------
+
+A frequent concern I hear is “you cannot trust LLMs” as they tend to
+hallucinate.  I hear this in particular when I explain that I use this as
+a replacement for Google a lot!  I approach this the same way I would when
+seeking advice from another human.  People can and are routinely wrong,
+and you learn to cross-check selectively.
+
+I treat AI as a collaborator or a pretty good intern.  You remain
+responsible for the final outcome.  The intern also happens to get better
+month by month as models improve.  And a bit like a human that digital
+intern has morals and wants to be argued with.  Some interns `don't want
+to talk about China
+<https://www.reddit.com/r/LocalLLaMA/comments/187oidh/deepseek_coder_7b_33b_thinks_its_trained_by_openai/>`__,
+other interns get a stroke `if you talk about certain people
```

Comment:
It might be more accurate to say "some interns don't know anything about some people" (which is totally true). I don't know that I've seen anyone totally breakdown at mentioning someone's name.

Reply:
Hah. Need to reword this. This was intended to be a joke at the AI interns, not real humans.